scripts/v1alpha3/create_cluster.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+METAL3_DIR="$(dirname "$(readlink -f "${0}")")/../.."
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+source "${METAL3_DIR}/lib/common.sh"
+
+# Cluster.
+CLUSTER_YAML=cluster.yaml
+
+
+make_cluster() {
+  envsubst < "${V1ALPHA3_CR_PATH}${CLUSTER_YAML}"
+}
+make_cluster | kubectl apply -n metal3 -f -
```

Comment:
same here, we should probably not duplicate all the scripts as for now the deployments for v1alpha2 and v1alpha3 are done the same way, just from different templates. That could be parameterized based on CAPI_VERSION

Reply:
The version independence of scripts and playbooks should be implemented in a separate PR. There's a risk that v1a2 CI runs break.